.github/workflows/codesonar.yaml

```diff
@@ -0,0 +1,68 @@
+name: CodeSonar
+on:
+  push:                  # Only run onces the Pull request is created
+    branches:
+      - 'release/**'               # Feature Branches
+  pull_request:
+    branches:
+      - 'release/**'       
```

Comment:
We don't use such branches. This looks like bad copy and paste from somewhere else.